### DISCUSSION TITLE
How do I add my own icons?

### DISCUSSION THREAD

**Discussion by SofiaRedmond** [zixpo/candybar#45] [Q&A]
What files should I modify to add my own icons?
 
➡️I understand that my icon should be saved in "drawable-nodpi" 

➡️And add it in the list in "xml/drawable.xml"

🚨The icon is applied but does not appear in the search results in the application, I would like to know which files I need to modify to add my own icons.

🙏 Please.

**Comment by sarsamurmu** [comment]
> What files should I modify to add my own icons?
>  
> ➡️I understand that my icon should be saved in "drawable-nodpi" 
> 
> ➡️And add it in the list in "xml/drawable.xml"
> 
> 🚨The icon is applied but does not appear in the search results in the application, I would like to know which files I need to modify to add my own icons.
> 
> 🙏 Please. 
> 

Make sure you are using the sample implementation. Also delete `xml-v26` directory, if there is any.

**Reply by SofiaRedmond** [reply]
Now is showing in list, but not appling into the launcher :(

**Reply by sarsamurmu** [reply]
Make sure you are adding mappings to `appfilter.xml` correctly. You can get app mappings by using the in-built icon request tool of the app.

**Reply by debz-g** [reply]
I faced a similar issue. The mapping is correct but it is still not applying.
